### PR TITLE
Add reference to signal postback tutorial

### PR DESCRIPTION
**Pull request by emdobrin**

### CODE REVIEW

File: resources/user-guides/signal-extension-and-rules-engine-integration.md [hunk 1]
```diff
@@ -12,11 +12,7 @@ To send PII data to external destinations, the `PII` action can trigger the Rule
 
 ## Rules tokens <a id="rules-tokens"></a>
 
-Rules tokens are special strings that are used in rule actions as values and are expanded by the SDK when the action is carried out. The format of a token is \`
-
-`, where token is any data element that is defined in Adobe Experience Platform Launch for a mobile property that identifies the source of the data from which the token is expanded. For example,`
-
-`can be used in the Signal postback action, where`My Data element for ECID\` is a data element that was created using the Mobile Core extension, and the data element type is Experience Cloud ID.
+Rules tokens are special strings that are used in rule actions as values and are expanded by the SDK when the action is carried out. The format of a token is `{%%token%%}`, where token is any data element that is defined in Adobe Experience Platform Launch for a mobile property that identifies the source of the data from which the token is expanded. For example, `{%%My Data element for ECID%%}` can be used in the Signal postback action, where `My Data element for ECID` is a data element that was created using the Mobile Core extension, and the data element type is Experience Cloud ID.
```
Comment (rekrishn) [inline resources/user-guides/signal-extension-and-rules-engine-integration.md]: Please change "Adobe Experience Platform Launch" to "Experience Platform Launch". 
(We only need to have the complete name at the 1st mention, and I'm sure it's been mentioned already!)

File: using-mobile-extensions/mobile-core/signals/README.md [hunk 2]
```diff
@@ -166,5 +168,5 @@ To send PII data to external destinations, the `PII` action can trigger the Rule
 
 For more information about `collectPii` and its usage, see [collectPii](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii).
 
-For more information about how to configure the Signal postbacks in Adobe Experience Platform Launch, see [doc link](https://aep-sdks.gitbook.io/docs/resources/user-guides/signal-extension-and-rules-engine-integration).
+For more information about how to configure the Signal postbacks in Adobe Experience Platform Launch, see [Signal extension and Rules Engine integration](https://aep-sdks.gitbook.io/docs/resources/user-guides/signal-extension-and-rules-engine-integration).
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signals/README.md]: Please change "Adobe Experience Platform Launch" to "Experience Platform Launch".

File: using-mobile-extensions/mobile-core/README.md [hunk 3]
```diff
@@ -22,9 +22,9 @@ Sessions contain information about the app's current lifecycle, such as the devi
...
 ## Rules Engine
 
-Mobile Core contains a rule engine that interprets all tracking, PII, Signals, _Request Content_, and extension-specific events for the SDK. Users can create rules, which when matched, trigger a _Response Content_ event for the listening extension to handle the associated consequence\(s\). For more information, see [Rules Engine](rules-engine/)
+Mobile Core contains a rule engine that interprets all tracking, PII, Signals, _Request Content_, and extension-specific events for the SDK. Users can create rules, which when matched, trigger a _Response Content_ event for the listening extension to handle the associated consequence\(s\). For more information, see [Rules Engine](rules-engine/).
 
-## Signals
+## Signal
 
-The Signals extension allows you to send data third-party endpoints via GET and POST requests. Signals are configured by using rules in Launch. For more information, see [Signals]().
+The Signal extension allows you to send data third-party endpoints via GET and POST requests. Signals are configured by using rules in Launch. For more information, see [Signal](signals/).
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/README.md]: Please change "Launch" to "Experience Platform Launch"

File: resources/user-guides/signal-extension-and-rules-engine-integration.md (hunk 1 above)
Comment (emdobrin) [reply resources/user-guides/signal-extension-and-rules-engine-integration.md]: Kept this one the same and renamed the next occurrence.

**Pull request merged**